packages/next/package.json

```diff
@@ -64,6 +65,7 @@
     "launch-editor": "2.2.1",
     "loader-utils": "1.1.0",
     "minimist": "1.2.0",
+    "mkdirp": "0.5.1",
```

Comment:
Either `mkdirp` or `mkdirp-then` can be removed. I'd vote for `mkdirp-then` and just using `promisify` 🤔